app/services/symbolService.js

```diff
@@ -105,7 +105,7 @@ class SymbolService {
     return { newExchangeSymbol };
   }
 
-  async getPrice(exchangeId = 'binance', symbol = 'BTC/USDT', startDate, endDate, interval = '1d', limit = 1, otherCurrency = 'CNY') {
+  async getPrice(startDate, endDate, exchangeId = 'binance', symbol = 'BTC/USDT', interval = '1d', limit = 1, otherCurrency = 'CNY') {
```

Comment:
This is incorrect. Changing the line will affect symbolController and priceScheduler

Reply:
> This is incorrect. Changing the line will affect symbolController and priceScheduler

Eslint throwed an error of "Default parameters should be last" here. I rolled it back to your version and leave it to you, hope you can fix that later.